main.tf

```diff
@@ -0,0 +1,215 @@
+module "label" {
+  source     = "github.com/cloudposse/terraform-terraform-label.git?ref=0.1.2"
+  attributes = "${var.attributes}"
+  delimiter  = "${var.delimiter}"
+  name       = "${var.name}"
+  namespace  = "${var.namespace}"
+  stage      = "${var.stage}"
+  tags       = "${var.tags}"
+}
+
+resource "aws_s3_bucket" "default" {
+  bucket = "${module.label.id}"
+  acl    = "private"
+  tags   = "${module.label.tags}"
+}
+
+resource "aws_iam_role" "default" {
+  name               = "${module.label.id}"
+  assume_role_policy = "${data.aws_iam_policy_document.assume.json}"
+}
+
+data "aws_iam_policy_document" "assume" {
+  statement {
+    sid = ""
+
+    actions = [
+      "sts:AssumeRole",
+    ]
+
+    principals {
+      type        = "Service"
+      identifiers = ["codepipeline.amazonaws.com"]
+    }
+
+    effect = "Allow"
+  }
+}
+
+resource "aws_iam_role_policy_attachment" "default" {
+  role       = "${aws_iam_role.default.id}"
+  policy_arn = "${aws_iam_policy.default.arn}"
+}
+
+resource "aws_iam_policy" "default" {
+  name   = "${module.label.id}"
+  policy = "${data.aws_iam_policy_document.default.json}"
+}
+
+data "aws_iam_policy_document" "default" {
+  statement {
+    sid = ""
+
+    actions = [
+      "ec2:*",
+      "elasticloadbalancing:*",
+      "autoscaling:*",
+      "cloudwatch:*",
+      "s3:*",
+      "sns:*",
+      "cloudformation:*",
+      "rds:*",
+      "sqs:*",
+      "ecs:*",
+      "iam:PassRole",
+    ]
+
+    resources = ["*"]
+    effect    = "Allow"
+  }
+}
+
+resource "aws_iam_role_policy_attachment" "s3" {
+  role       = "${aws_iam_role.default.id}"
+  policy_arn = "${aws_iam_policy.s3.arn}"
+}
+
+resource "aws_iam_policy" "s3" {
+  name   = "${module.label.id}-s3"
```

Comment:
The problem with taking this shortcut is we don't honor the `delimiter`. Should therefore create new label.